packages/driver/cypress/e2e/e2e/origin/snapshots.cy.ts

```diff
@@ -2,107 +2,153 @@
 import '../../../support/utils'
 
 describe('cy.origin - snapshots', { browser: '!webkit' }, () => {
-  const findLog = (logMap: Map<string, any>, displayName: string, url: string) => {
-    return Array.from(logMap.values()).find((log: any) => {
-      const props = log.get()
+  it('does not create snapshots after the document has unloaded and the AUT has navigated cross-origin', () => {
+    cy.visit('/fixtures/generic.html')
+    cy.visit('http://www.foobar.com:3500/fixtures/generic.html')
+    cy.then(() => {
+      const snapshot = cy.createSnapshot()
 
-      return props.displayName === displayName && (props?.consoleProps?.URL === url || props?.consoleProps()?.URL === url)
+      expect(snapshot).to.be.null
     })
-  }
-  let logs: Map<string, any>
+  })
+
+  it('takes snapshots from the secondary origin even after the primary AUT has been unloaded from state', () => {
+    const findLog = (logMap: Map<string, any>, selector) => {
+      return Array.from(logMap.values()).find((log: any) => {
+        const props = log.get()
 
-  beforeEach(() => {
-    logs = new Map()
+        return (props?.consoleProps?.Selector === selector)
```

Comment:
What log is this associated with?

Reply:
In this case ,we are just digging out the `[data-cy="assertion-header"]` get log to make sure the snapshot is sent back over and not omitted with our new logic changes

Reply:
Is that associated with the origin command log or an xhr log?

Reply:
origin command log since the get is invoked inside the cy.origin